libcontainer/process_linux.go

```diff
@@ -634,6 +719,31 @@ func (p *initProcess) forwardChildLogs() chan error {
 	return logs.ForwardLogs(p.logFilePair.parent)
 }
 
+func sendContainerProcessState(listenerPath string, state *specs.ContainerProcessState, fd int) error {
+	conn, err := net.Dial("unix", listenerPath)
+	if err != nil {
+		return fmt.Errorf("cannot connect to %q: %v\n", listenerPath, err)
```

Comment:
1. `\n` should be removed.
2. Please to use %w for error.

Reply:
3. I think `net.Dial` already includes path/address in the error message, as well as the error (`dial`), so there's no need to wrap it.

Reply:
Cool, returning err then

Reply:
> I think net.Dial already includes path/address in the error message, as well as the error (dial), so there's no need to wrap it.

Correct:

```
$ cat main.go 
package main

import (
	"fmt"
	"net"
)

func main() {
	_, err := net.Dial("unix", "/fdsjkfhdjk")
	if err != nil {
		fmt.Printf("%s\n", err)
	}
}
$ go run main.go 
dial unix /fdsjkfhdjk: connect: no such file or directory
```